sdk/template/template/package.json

```diff
@@ -19,13 +18,23 @@
       }
     ]
   },
+  "types": "types/latest/template.d.ts",
+  "typesVersions": {
+    "<3.6": {
+      "types/latest/*": [
+        "types/3.1/*"
+      ]
+    }
+  },
```

Comment:
I see the following for storage-queue.
![image](https://user-images.githubusercontent.com/10452642/109551760-d0f6b400-7a85-11eb-9bee-fffee0ef4a0d.png)

Does your change works the same?

Reply:
The hierarchy is the same.